ported/omb/rdm_latency_threaded

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
```

Comment:
Yes. This is to provide backward compatibility with our existing tests by taking the modified test and the current set of loops and skip settings and embedding them in the script. This should provide a transparent change to the existing architecture while providing the enhancements with no discernible difference. If we should change the defaults in rdm_pingpong, this test will remain the same because we pass the loops and skip variables by hand prior to handling any other passed arguments.